### ISSUE TITLE
IOS 13.3 checkra1n Ota error bootstrap

### ISSUE THREAD

**Issue by djshadeemotiveproductions** [checkra1n/BugTracker#688]
Hi I’m on iOS 13.3
Checkra1n 0.9.7 
Tried jailbreaking it went through now when installing Cydia it pops up saying ota bootstrap error. Remove ota and reboot. 
Its a fresh install of 13.3 
I’m on iPhone 6s unlocked game 64gb. 

I tried regular and a different boot no luck. And there is no updates. Please help

**Comment by shtuseriff** [comment]
Did you ever figure this out? I have the same issue. iPhone X, just upgraded to 13.3 from 12.1.2, and there is no OTA in the Storage section in the Settings app.

**Comment by djshadeemotiveproductions** [comment]
Yeah i just did a restore for the iphone clean reset and then re jailbroken it. Ig when being a software developer it kinda messes with the files
